pkg/controller/provider/container/openstack/client.go

```diff
@@ -126,15 +126,20 @@ func (r *Client) Authenticate() (err error) {
 		} else {
 			cacert := []byte(r.getStringFromSecret(CACert))
 			roots := x509.NewCertPool()
-			ok := roots.AppendCertsFromPEM(cacert)
-			if !ok {
-				r.Log.Info("the CA certificate is malformed or was not provided, falling back to system CA cert pool")
+
+			if len(cacert) == 0 {
+				r.Log.Info("CA certificate was not provided, falling back to system CA cert pool")
```

Comment:
then it's not a fall back, how about
```suggestion
				r.Log.Info("CA certificate was not provided, system CA cert pool is used")
```

Reply:
done